submissions/Lu-sure/Popup/style.css

```diff
@@ -0,0 +1,207 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: content-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+}
+
+.navigation {
+  display: flex;
+  justify-content: flex-end;
+  height: 65px;
+  align-items: center;
+}
+
+ul {
+  list-style: none;
+}
+
+.nav-icon {
+  display: flex;
+  align-items: center;
+}
+
+.nav-text-item {
+  text-decoration: none;
+  color: #5f5f5f;
+  margin-right: 25px;
+}
+
+.nav-text-item:hover, .popup-bottom:hover {
+  text-decoration: underline;
+}
+
+.nav-img-item, .menu-img {
+  height: 32px;
+  width: 32px;
+}
+
+.nav-img-item {
+  margin-right: 25px;
+}
+
+.menu-item-border {
+  border: 3px solid transparent;
+  border-radius: 8%;
+}
+
+.menu-item-border:hover {
+  border-color: rgb(145, 181, 248);
+}
+
+.bell-item {
+  width: 26px;
+  height: 30px;
+}
+
+.bell-img {
+  width: 100%;
+  height: 100%;
+}
+
+.round-img {
+  border-radius: 50%;
+}
+
+#hoogle-menu, #more-button {
+  opacity: 0;
+  position: absolute;
+  z-index: -1;
+}
+
+ /* ---------------- popup ---------------- */
+
+ .popup {
+  overflow-y: auto;
+  padding-top: 8px;
+  width: 100%;
+  height: 550px;
+}
+
+.popup-wrapper {
+  width: 370px;
+  border: 1px solid #cdcdcd;
+  box-shadow: -5px 5px 15px 10px rgba(0, 0, 0, 0.1);
+  position: absolute;
+  top: 70px;
+  right: 70px;
+  display: none;
+}
+
+/*  ----- grey background & white foreground triangle ----- */
```

Comment:
Avoid a comment

Reply:
I want to clarify. Comments are needed in a place where there is some not obvious behavior. But in this case, yes, comments are optional.